addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/ApplicationErrorScanRuleUnitTest.java

```diff
@@ -347,6 +350,101 @@ void shouldNotRaiseAlertForResponseCodeOkAndContentTypeWebAssemblyWhenFilePayloa
         assertThat(alertsRaised.size(), equalTo(0));
     }
 
+    private void createHttpMessageWithContentTypeAndCustomEvidence(
+            HttpMessage msg, String contentType, String errorString)
+            throws HttpMalformedHeaderException {
+        msg.setRequestHeader(REQUEST_HEADER);
+        msg.setResponseBody(createResponseHeader(OK));
```

Comment:
I guess this meant to be `setResponseHeader`.

Reply:
Good catch

Reply:
oops, my bad. I guess it was still working fine because maybe it gives a response code of 200 by default, if not defined.